src/Core/src/Platform/Android/MauiApplication.cs

```diff
@@ -72,8 +52,11 @@ public override void OnConfigurationChanged(Configuration newConfig)
 
 		public class ActivityLifecycleCallbacks : Java.Lang.Object, IActivityLifecycleCallbacks
 		{
-			public void OnActivityCreated(Activity activity, Bundle? savedInstanceState) =>
+			public void OnActivityCreated(Activity activity, Bundle? savedInstanceState)
+			{
+				activity.RegisterComponentCallbacks(new ActivityComponentCallbacks(activity));
```

Comment:
Should there be a matching UnregisterComponentCallbacks in the activity destroyed?